packages/platforms/react-native/lib/id-generator.ts

```diff
@@ -15,12 +15,13 @@ const idGenerator: IdGenerator = {
   generate (bits: BitLength): string {
     const bytes = new Uint8Array(bits / 8)
```

Comment:
This `Uint8Array` is also unnecessary since we only use it for the `length`, which is always the value we pass to the constructor:

```js
> const bits = 128
>
> new Uint8Array(bits / 8).length
// => 16
> bits / 8
// => 16
```

```suggestion
    const bytes = bits / 8
```